### ISSUE TITLE
ob-close no documentation

### ISSUE THREAD

**Issue by ice-blaze** [oblique-bit/oblique#38]
There is no documentation on how to have the close button for an alert https://oblique.bit.admin.ch/components/alert?theme=material `class="ob-close"` was one way to get it.

**Comment by gillerr** [comment]
Actually alerts have been turned into an angular component and as such are not meant to be closed. 

An Angular component cannot remove itself, only its parent can do that. An an alert with a parent component that manage it, is quite exactly what the notification component is. This means a closable alert actually is a notification. 

While you can have multiple notification containers on the same page, all notifications will slide in and be absolutely positioned. 
Should you use a notification that behaves like the alerts did, please do state it and we'll consider implementing it.

Anyway, the documentation is misleading and will be fixed

**Comment by gillerr** [comment]
The documentation has been fixed with 7.0.3. Moreover, we did integrate `.ob-close` CSS in `oblique-alerts.css. Note that this file is intended to be used when alerts aren't used with Angular. There's no guarantee that this file will stay in Oblique forever.

Anyway, this snippet will allow you to include close buttons in alerts:
```
	.ob-alert .ob-close {
		float: right;
		padding: 0;
		font-size: 1.5rem;
		font-weight: 700;
		line-height: 1;
		border: 0;
		color: inherit;
		background-color: transparent;
		opacity: .5;
		cursor: pointer;

		&:hover {
			opacity: 1;
		}
	}
```

**Comment by gillerr** [comment]
@ice-blaze , I'll close this issue. If you need angular Alerts that are closable, please do open a new ticket and we'll adapt the notification component for that use case.
